assets/section-main-product.css

```diff
@@ -554,6 +590,28 @@ a.product__text {
   width: 100%;
 }
 
+.product__media-toggle:focus-visible {
+  outline: 0;
+  box-shadow: none;
+}
+
+.product__media-toggle.focused {
+  outline: 0;
+  box-shadow: none;
+}
+
+.product__media-toggle:focus-visible:after {
+  outline: 0.2rem solid rgba(var(--color-foreground),.5);
+  outline-offset: 0.3rem;
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),0 0 0.5rem 0.4rem rgba(var(--color-foreground),.3);
+}
+
+.product__media-toggle.focused::after {
+  outline: 0.2rem solid rgba(var(--color-foreground),.5);
+  outline-offset: 0.3rem;
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),0 0 0.5rem 0.4rem rgba(var(--color-foreground),.3);
+}
```

Comment:
We can't really combine these. The `.focused` class is only available when `:focus-visible` is not supported by the browser. And if the browser does not recognize part of a selector in CSS, it will ignore the entire group of selectors.

I'll update the spacing after commas and background color, though.

Reply:
ah yeah right! Forgot about this as a reason 👍